src/lib/crypt_ops/aes_openssl.c

```diff
@@ -111,19 +61,22 @@ aes_new_cipher(const uint8_t *key, const uint8_t *iv, int key_bits)
   EVP_EncryptInit(cipher, c, key, iv);
   return (aes_cnt_cipher_t *) cipher;
 }
+/**
```

Comment:
Nitpick: newline?
And for the other functions following this one.

Reply:
Done in 3acadb6d890391d547252a282918ea1bab0e2078